### PR TITLE
Increases the basic required version of php

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Increases the basic required version of php

### Why is it needed?

https://docs.modx.com/3.x/en/getting-started/server-requirements

The documentation states that minimum version = 7.1. This fix upgrades the version to the required

https://github.com/modxcms/revolution/blob/3.x/.github/workflows/ci.yml#L28 - Do we need tests for 7.0?

### Related issue(s)/PR(s)
NA

### CODE REVIEW

**Comment by sergant210** [comment]
7.1 and 7.2 are not supported now. [Supported versions](https://www.php.net/supported-versions.php).

**Comment by Ibochkarev** [comment]
> 7.1 and 7.2 are not supported now. [Supported versions](https://www.php.net/supported-versions.php).

I can change to 7.3 and remove the old version of php in the ci.yml, but this needs the approval of the integrators

**Comment by opengeek** [comment]
We will not be removing support for 7.1/7.2 at this time. They are still in widespread use regardless of what the core PHP maintainers say.

**Comment by opengeek** [comment]
This needs the check for 7.0 removed.

**Review by opengeek — CHANGES_REQUESTED** [review]
This needs the check for 7.0 removed.

**Comment by Ibochkarev** [comment]
> This needs the check for 7.0 removed.

I will submit a commit with the 7.0 checkout removal in the Github action tests

**Review by opengeek — APPROVED** [review]

**Comment by JoshuaLuckers** [comment]
Don't forget to change it here: https://github.com/modxcms/revolution/blob/0acc0d92fcea3fc97ab5692a8d3b55648fd6ef2c/setup/provisioner/requirements.php#L11

**Comment by opengeek** [comment]
Why do we have all of this duplicated there?

**Comment by opengeek** [comment]
What I mean to say is, I wonder if we can find a way to use the composer declarations in setup rather than duplicating it all.

**Comment by JoshuaLuckers** [comment]
> What I mean to say is, I wonder if we can find a way to use the composer declarations in setup rather than duplicating it all.

Yes that would be great, but is the composer.json in the downloadable build?

**Comment by opengeek** [comment]
> > What I mean to say is, I wonder if we can find a way to use the composer declarations in setup rather than duplicating it all.
> 
> Yes that would be great, but is the composer.json in the downloadable build?

It certainly should be.

**Comment by JoshuaLuckers** [comment]
> > > What I mean to say is, I wonder if we can find a way to use the composer declarations in setup rather than duplicating it all.
> > 
> > 
> > Yes that would be great, but is the composer.json in the downloadable build?
> 
> It certainly should be.

I've downloaded the alpha-2 release from [modx.com/download](http://modx.com/download), composer.json is not included.

**Pull request merged**
